tools/autograd/derivatives.yaml

```diff
@@ -181,6 +181,15 @@
 - name: all.dim(Tensor self, int dim, bool keepdim=False) -> Tensor
   self: not_implemented("all")
 
+- name: acosh(Tensor self) -> Tensor
+  self: grad * (self.pow(2) - 1).rsqrt()
+
+- name: asinh(Tensor self) -> Tensor
+  self: grad * (self.pow(2) + 1).rsqrt()
+
+- name: atanh(Tensor self) -> Tensor
+  self: grad * 1 / (-self.pow(2) + 1)
```

Comment:
nit: `1 - XXX` ?

Reply:
The only reason I didn't use `1 - xxx` (which seems better to the eyes), is that other functions use `-xxx + 1` format. Still, will replace to `1 - xxx1. Thanks @albanD!

Reply:
Should be resolved here: https://github.com/pytorch/pytorch/pull/38388/commits/d66d7242ec60544d468ef99dcc78d63628d84723. Thanks!